### PR TITLE
IOSAS - Initial Navigation 

### PR DESCRIPTION
**Pull request by mmumby**
- Updated Unauthenticated view to include a create EOI button, changed layout of login components. 
- Added EOI and application tables to homepage with some mock data, and mock alerts.
- Added profile route - WIP for now.
- Added EOI form with all field inputs from pdf
- Added school application forms from pdf (Will be broken out into tabs in future tickets)

- Removed routes from cloned edx application
- Made all existing pages responsive

### CODE REVIEW

File: frontend/src/components/Login.vue [hunk 1]
```diff
@@ -1,82 +1,113 @@
...
-            class="my-1"
-            align="center"
-            justify="center"
+  <v-container fluid class="content-container">
+    <div class="d-flex justify-center">
+      <v-row justify="center">
+        <v-col cols="12" sm="12" md="6" xs="12">
+          <v-card title="Log in with your primary Business BCeID">
+            <div class="card-content">
+              <p>Enter your user ID and password to contine</p>
```
Comment (fwpushan) [inline frontend/src/components/Login.vue]: Please fix typo

**Review by fwpushan — COMMENTED** [review]
All good, one typo

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+# dependencies
+/node_modules
+package-lock.json
+
+# tools
+/tools
```
Comment (fwpushan) [inline .prettierignore]: includes tools also

**Review by fwpushan — COMMENTED** [review]
Please check

File: frontend/src/components/Login.vue (hunk 1 above)
Comment (mmumby) [reply frontend/src/components/Login.vue]: good eye, fixed.

File: .prettierignore (hunk 2 above)
Comment (mmumby) [reply .prettierignore]: removed

**Review by fwpushan — APPROVED** [review]
All good

**Pull request merged**